release/models/platform/openconfig-platform-types.yang

```diff
@@ -218,6 +224,28 @@ module openconfig-platform-types {
     uses oc-types:min-max-time;
   }
 
+  grouping component-last-poweroff-reason {
+    description
+      "Common grouping for recording the reason of a component's
+      power-off state";
+
+    leaf trigger {
+      type component-last-poweroff-reason-trigger;
```

Comment:
this type is not defined.  Did you want to do something like:

      type identityref {
        base COMPONENT_POWEROFF_REASON;
      }